.env_sample

```diff
@@ -9,4 +9,7 @@ export SECRET_KEY=''
 
 
 export RECAPTCHA_PUBLIC_KEY=''
-export RECAPTCHA_PRIVATE_KEY=''
\ No newline at end of file
+export RECAPTCHA_PRIVATE_KEY=''
+export FITBIT_CLIENT_ID=22DRQ3
+export FITBIT_CLIENT_SECRET=a9c286f26f3228fc572a53b93bea5b73
+export ROOT_URL='http://127.0.0.1:8000'
```

Comment:
It is good practice to have sensitive information such as the keys not visible on the codebase. I would advise having this on the manually configurable env settings on both travis and heroku.

Reply:
This is a sample .env and the secret codes are not real or valid. They are supposed to show what is to be added to the real .env